frontend/src/components/MeetingRoom/CaptionsButton/CaptionsBox/CaptionsBox.tsx

```diff
@@ -0,0 +1,71 @@
+import { ReactElement } from 'react';
+import { Box } from '@mui/material';
+import SingleCaption from './SingleCaption';
+import { SubscriberWrapper } from '../../../../types/session';
+
+export type CaptionsBoxType = {
+  subscriberWrappers: SubscriberWrapper[];
+  isCaptionsEnabled: boolean;
+  isMobileView: boolean;
+  localPublisherCaptions?: string | null;
+};
+
+/**
+ * CaptionsBox Component
+ *
+ * This component shows a list of the captions that are currently in the meeting room.
+ * @param {CaptionsBoxType} props - the props for the component.
+ *  @property {SubscriberWrapper[]} subscriberWrappers - an array of subscriber wrappers.
```

Comment:
Missing some props

Reply:
oh man, good catch with these. I wish there was an automated linter of sorts that would catch these. I wrote the docs before I added the other properties, same with the other cases that I missed